test/carts/postPaymentIT.js

```diff
@@ -71,7 +71,7 @@ describe('Magento postPayment', function () {
                 .post(env.cartsPackage + 'postCartEntry')
                 .query({
                     currency: 'USD',
-                    quantity: 5,
+                    quantity: 1,
```

Comment:
In the PR comment you mention having to fix/change some quantity for tests to pass, is this what you're referring to? I agree with Mark that this looks like a workaround to "fix" another problem we might have, either a bug in the Magento code or in the way the tests are running ... I'd rather investigate ti understand why this needs to be changed.

Reply:
@herzog31 passes on master because I changed in magneto instances. 
@cjelger pls note that I have an issue for this https://jira.corp.adobe.com/browse/CIF-399. I didn't want to take fix it now because the PR is way too big.

Reply:
The available product quantity.

Reply:
I assumed you increased the available product quantity in Magento, so that there are enough products available for the tests to add them to carts?